src/rendering/screenspacerenderable.cpp

```diff
@@ -522,18 +533,60 @@ glm::mat4 ScreenSpaceRenderable::scaleMatrix() {
 
     glm::mat4 scale = glm::scale(
         glm::mat4(1.f),
-        glm::vec3(_scale, _scale * textureRatio, 1.f)
+        glm::vec3(_scale, textureRatio*_scale, 1.f)
     );
+  
+    return scale;
+}
+glm::vec2  ScreenSpaceRenderable::screenSpacePosition() {
+    return glm::vec2(_cartesianPosition.value().x, _cartesianPosition.value().y);
```

Comment:
I think just `return glm::vec2(_cartesianPosition)` should be fine.

Reply:
![image](https://user-images.githubusercontent.com/24265984/156612539-3e58311a-75ca-4b48-a43e-b080d7d676a5.png)

![image](https://user-images.githubusercontent.com/24265984/156612515-0010f6d2-5867-4e33-9cd1-f1b3be320af6.png)

Reply:
As I was wrong about the "value" not being needed, what about this: `return glm::vec2(_cartesianPosition.value())`

That should be calling the vec2 constructor with a vec3 :)

Reply:
Works!:)